lib/ui/view/nav_bar/nav_bar_viewmodel.dart

```diff
@@ -1,7 +1,9 @@
+import 'package:flutter/material.dart';
 import 'package:stacked/stacked.dart';
 
 class NavBarViewModel extends IndexTrackingViewModel {
   static final NavBarViewModel instance = NavBarViewModel._internal();
+  List<BottomNavigationBarItem> bottomNavList = [];
```

Comment:
I don't think there is need refactoring this to be in the viewmodel, It has already break a rule, that ensure material or cupertino widget is not imported into the VM. Please make the changes in the view.

Reply:
Done